auth/src/main/java/org/jboss/pnc/bacon/auth/model/CacheFile.java

```diff
@@ -75,9 +75,9 @@ public static void writeCredentialToCacheFile(String keycloakUrl, String realm,
     }
 
     private static void createConfigFolderIfAbsent() {
-        if (!fileExists(Constant.CONFIG_FOLDER)) {
+        if (!fileExists(Constant.DEFAULT_CONFIG_FOLDER)) {
```

Comment:
It should create the folder where the saved-user.json will be actually stored, not the default location.